content/en/docs/reference/glossary/multicluster-mesh.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Multicluster Mesh
```

Comment:
Interesting take, but it seems to me the term multicluster is thrown around and used so frequently that we need to concisely define what that is.

Reply:
I think the term `Multicluster` really only applies to a single mesh which spans multiple clusters. So `Multicluster` and `Multicluster Mesh` are the same thing.

If you have two or more meshes, the cluster topology is irrelevant. That's `Mesh Federation` (or something mesh-focused, rather than cluster-focused).

Reply:
You've hit the nail right on the head. I'm trying to differentiate between 1) multiple clusters that are contributing to a single mesh vs. 2) multiple clusters that are each configured as single meshes and then federated. Both of these involve multiple clusters, so I think we need to be clear that when we use the term multicluster we mean case 1) and I think that consistently using the explicit term "multicluster mesh" in the docs is best.

Reply:
Ok that's fair. But it still doesn't solve the problem of folks using the term `multicluster`. I think if we define this, we should also define `multicluster` and reference `multicluster mesh`. This makes it clear that WRT Istio, they're the same thing.

Reply:
+1 to documenting a definition for this term; it may or may not help minimize the times we have to ask "by multi-cluster, do you mean one mesh or multiple?", but it can't hurt.

Reply:
Maybe we should add a glossary entry for multicluster by itself, and not for "multicluster mesh"?

Then under that we can say that multiple clusters can be part of a single mesh, or they can be part of multiple meshes and communicate using federation.

Reply:
We could have a glossary entry for "Multicluster" only which just means more than one cluster running Istio, and explain that it can be modeled as a single mesh or multiple meshes that are federated. But, if we also remove the entry for "service mesh", we have no concrete term to use when talking about the single mesh composed from multiple clusters topology. Nathan and I have been leaning in the opposite direction, i.e., define the term "Multicluster" as the single mesh case only. Mesh Federation is then defined as the multiple mesh case.

Maybe defining all three terms in the glossary would be best: Multicluster Mesh (single mesh), "Mesh Federation" (multiple mesh), and just "Multicluster" (defined as can be either a "Multicluster Mesh" or a "Mesh Federation".

Reply:
PTAL at PR #4813 
IMO, the content there could help move this conversation forward.